locales/en/app.ftl

```diff
@@ -821,6 +821,16 @@ meta-desc = Find out if you’ve been part of a data breach with {-brand-fx-moni
 brand-fx-monitor = {-brand-fx-monitor}
 sign-in = Sign In
 
+## User menu
+
+menu-button-title = User menu
+menu-button-alt = Open user menu
+menu-item-fxa = Manage your {-brand-firefox} account
```

Comment:
This should use the term for Firefox account
```suggestion
menu-item-fxa = Manage your {-brand-fx-account}
```

You probably want to move the definition and comment around line 815.